docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,69 @@
+
+For temporary job files, you can utilize ``/scratch/<username>``. Please be mindful of the ``/scratch`` purge policy, which is can be found on the `Research Computing Policy Page <https://rc.northeastern.edu/policy/>`_.
+
+Clean ~/.conda directory
+=======================================================
+.. note::
+  Conda environments are part of your research and should be stored in your PI's ``/work`` directory. 
+
+If conda environments are stored in your ``/home/<username>`` directory, here are some suggestions to reduce storage size of the environments.
+
+To remove any unused packages and caches from using conda, from an srun session, ``srun --pty /bin/bash``, load the version of anaconda/minicoda the environment are built with and run: ::
+
+ source activate <your environment>
+ conda clean --all
+
+This will only delete unused packages in your ``~/.conda/pkgs`` directory.
+
+To remove any unused conda environments, run: ::
+
+ conda env list
+ conda env remove --name <your environment>
+
+Clean ~/.singularity directory
+=======================================================
+If you have pulled any containers to Discovery using Singularity, you can clean your container cache in your ``/home/<username>`` directory by running the following command from an srun session (``srun --pty /bin/bash``): ::
+
+ module load singularity/3.5.3
+ singularity cache clean all
+
+To avoid your ``~/.singularity`` directory filling up, you can set a temporary directory for when you pull containers to store the cache in that location; an example of this procedure, (where ``<project>`` is your PI's ``/work`` directory) is: ::
+
+ mkdir /work/<project>/singularity_tmp
+ export SINGULARITY_TMPDIR=/work/<project>/singularity_tmp
+
+Then, use singularity to pull your container as you normally would. 
+
+Best practices
+=======================================================
+The following sections provide best practices to maintain your ``/home/<username>`` quota for work that utilizes conda environments and containers.
+
+Conda environments
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Conda virtual environments should be used for all python based workflows and should be located in your PI's /work directory. To create a conda environment in a ``/work`` directory, please use the ``--prefix`` flag with conda as follows (where ``<project>`` is your PI's ``/work`` directory and ``<my directory>`` is an empty directory to store your conda environment): ::
+
+ conda create myenv --prefix=/work/<project>/<my directory>
+
+More information about creating custom conda environments can be found here :ref:`conda`
+
+Conda environments that are stored in a commonly accessibly directory in your PI's ``/work`` directory will be accessible by fellow group members will be able to ``source activate <env name>`` and utilize the same conda environment to save storage space and time in building duplicate conda environments. 
```

Comment:
This paragraph is a little confusing, I wasn't sure exactly how to clean it up. (looks like you just changed your mind about how to write it and missed deleting some of it)